README.md

```diff
@@ -297,6 +297,33 @@ make load-test
 - Review logs that will be sent to Log Analytics when configured
   - See `deploy/loganalytics` for directions
 
+## How Codespaces is built?
+
+A development container is a fully-featured development environment running in a container. The development container is a Docker container running locally or remotely that, at a high-level, contains the following:
```

Comment:
I would focus specifically on Codespaces - you might briefly mention dev containers (a bullet point or sentence)